device.go

```diff
@@ -340,7 +340,11 @@ type BlockDevice struct {
 	DeviceType string
 	DeviceInfo DeviceInfo
 
-	// Path at which the device appears inside the VM, outside of the container mount namespace.
+	// SCSI Address of the block device, in case the device is attached using SCSI driver
+	// SCSI address is in the format SCSI-Id:LUN
+	SCSIAddr string
```

Comment:
Don't you think this should be part of the `DeviceInfo` ?

Reply:
No the DeviceInfo includes fields that are common to all devices. This is specific to Block device.

Reply:
Makes sense